example-client/client.py

```diff
@@ -24,26 +47,44 @@ def run_structure(input: str) -> Optional[str]:
     Returns:
         The output of the Structure program.
     """
-    # Start a run with the args "What is 5 ^ 2".
+    # Start a run with the input args.
```

Comment:
do we show an example anywhere of what the input args should look like?

Reply:
I mean, I see them down below - in the run - but wondering if we want to explain in more detail what they can be

Reply:
verify latest